flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogDatabase.java

```diff
@@ -18,270 +18,241 @@
  */
 package org.apache.iceberg.flink;
 
-import java.io.File;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assumptions.assumeThat;
+
+import java.nio.file.Path;
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import org.apache.flink.table.catalog.exceptions.DatabaseNotEmptyException;
 import org.apache.flink.types.Row;
+import org.apache.iceberg.Parameters;
 import org.apache.iceberg.Schema;
 import org.apache.iceberg.catalog.Namespace;
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.types.Types;
 import org.assertj.core.api.Assertions;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Assume;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.TestTemplate;
 
-public class TestFlinkCatalogDatabase extends FlinkCatalogTestBase {
+public class TestFlinkCatalogDatabase extends CatalogTestBase {
 
-  public TestFlinkCatalogDatabase(String catalogName, Namespace baseNamespace) {
-    super(catalogName, baseNamespace);
+  @Parameters(name = "catalogName={0}, baseNamespace={1}")
+  static List<Object[]> parameters() {
```

Comment:
I think it was correct the way it was before (by having it defined in `CatalogTestBase`). Subclasses that need different parameterization will just override this method. That's also how it is done in https://github.com/apache/iceberg/pull/9368

Reply:
ack